doc/tutorials/acrn_configurator_tool.rst

```diff
@@ -328,7 +335,10 @@ Save and Check for Errors
       :align: center
```

Comment:
Lines 244/245  or 251/252  consider changing "scenario configuration files for a former release may not work for a latter if they are not upgraded." to "scenario configuration files for a former release may not work in the current release unless they are upgraded."